src/components/info-button/info-button.jsx

```diff
@@ -26,7 +26,7 @@ class InfoButton extends React.Component {
     }
     render () {
         const messageJsx = this.state.visible && (
-            <div className="info-button-message">
+            <div className="info-button-message validation-info">
```

Comment:
Are you adding validation info to pick up the styles from https://github.com/LLK/scratch-www/blob/5ab2bf100b59123c41654dca76c615f6b4046002/src/components/forms/validation-message.scss#L82

I don't love that because if I change the validation-info class, the changes I'm going to know that I also need to check the InfoButton are pretty small.  Also, is the css compiler smart enough not to rename them differently given there's no require here?

Reply:
Yeah, you're right, this is not the right way to do it! Even if it works in practice...

Take a look now.